### PR TITLE
[agw][lte] Logging enhancement

### PR DESCRIPTION
**Pull request by pruthvihebbani**
[agw][lte] Logging enhancement

## Summary

Added IMSI/ue-id to the logs for better tracing and changed log levels to WARNING/INFO which are part of the expected call flow. Issue id - https://github.com/magma/magma/issues/7097

## Test Plan

- Verified sanity suite

### CODE REVIEW

File: lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c [hunk 1]
```diff
@@ -1051,10 +1058,17 @@ int mme_app_handle_create_sess_resp(
...
+      "id " MME_UE_S1AP_ID_FMT "\n",
+      ue_context_p->mme_ue_s1ap_id);
+
   OAILOG_DEBUG_UE(
       LOG_MME_APP, ue_context_p->emm_context._imsi64,
-      "MME S11 teid = %u, cause = %d, ue_id = %u\n", create_sess_resp_pP->teid,
-      create_sess_resp_pP->cause.cause_value, ue_context_p->mme_ue_s1ap_id);
+      "MME S11 teid = %u, cause = %d, ue_id = " MME_UE_S1AP_ID_FMT "\n",
+      create_sess_resp_pP->teid, create_sess_resp_pP->cause.cause_value,
+      ue_context_p->mme_ue_s1ap_id);
```
Comment (ulaskozat) [inline lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c]: why don't we combine these?

Comment (ardzoht) [reply lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c]: +1 on this, let's use INFO and combine the information logged here

**Review by ardzoht — APPROVED** [review]
lgtm, please address Ulas' comment before merging

**Review by ulaskozat — APPROVED** [review]
LGTM

**Pull request merged**